kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/security/auth/ResourceType.java

```diff
@@ -34,6 +34,13 @@
      * A Pulsar topic.
      */
     TOPIC((byte) 2),
+
+    /**
+     * A Pulsar namespace.
+     */
+    NAMESPACE((byte) 3),
```

Comment:
It seems that in this PR there's no place that a `Resource` object's type is `NAMESPACE`? If so, we should remove this change and add it in the followed PR.